Meta/Azure/nightly-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+# Pipeline definition for nightly or other scheduled jobs we don't want to run on every PR in Azure
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
```

Comment:
Wouldn't it be good if this CI job was specifically called "Daily midnight coverage build" so we instantly recognize its purpose?

Reply:
I don't think so, we might want to add more nightly or weekly jobs like port building in the future.

Reply:
Alright then, you can resolve this.